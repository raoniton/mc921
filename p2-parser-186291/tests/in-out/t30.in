class Main {
    public int f(int n, int k) {
        int p, q, t;
        if (n < 2) {
            k = 0;
            return n;
        }
        else {
            t = f(n-1, p) + f(n-2, q);
            k = p + q + 1;
            return t;
        }
    }

    public static void main(String[] args) {
        Main obj = new Main();
        int m = 9;
        print(obj.f(3, m), m);
    }
}