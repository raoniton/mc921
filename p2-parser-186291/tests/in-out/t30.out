ParserError: Before ( @ 9:18
